src/main/java/com/nirmata/workflow/details/internalmodels/StartedTask.java

```diff
@@ -23,11 +23,13 @@
 {
     private final String instanceName;
     private final LocalDateTime startDateUtc;
+    private int progress;
```

Comment:
Since we provide setter to let task update its value, we cannot define it as final here. Its value is subject to be changed.